lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -4712,7 +4714,7 @@ def get_multi_check_constraints(
                 "sqltext": sqltext,
                 "comment": comment,
             }
-            if m and m.group(2):
+            if m and " NOT VALID" in m.groups():
                 entry["dialect_options"] = {"not_valid": True}
```

Comment:
**Federico Caselli** (CaselIT) wrote:

let's also add a flag for no inherit

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5083

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5083